Formula/verovio.rb

```diff
@@ -0,0 +1,22 @@
+class Verovio < Formula
+  desc "Command-line MEI music notation engraver"
+  homepage "https://www.verovio.org"
+  url "https://github.com/rism-digital/verovio/archive/refs/tags/version-3.12.1.tar.gz"
+  sha256 "7f69b39e25f9662185906c9da495437e09539a520b9dda80f1bef818e905881b"
+  license "LGPL-3.0-only"
+  head "https://github.com/rism-digital/verovio.git", branch: "develop"
+
+  depends_on "cmake" => :build
+
+  def install
+    cd "tools" do
+      system "cmake", "../cmake", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    system bin/"verovio", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.